v3api/transcripts.go

```diff
@@ -58,11 +58,18 @@ type CancelObjectRepresentation struct {
 	Success bool `json:"success"`
 }
 
+// CallParams contains the call params the caller wants to override
+type CallParams struct {
```

Comment:
Definitely for the future, but can leverage this struct to have a ``Context`` parameter too, just to allow people to provide their context for request cancellation.